pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -130,6 +136,8 @@ type RunnerInfo struct {
 	Command []string `json:"command,omitempty"`
 	//ImagePullPolicy for runner pod
 	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
+	// Annotations that needs to be provided in the pod for pod that is getting created
+	AppAnnotations []ApplicationAnnotations `json:"appannotations"`
```

Comment:
Can we use this as map[string]string while defining here (and in the ExperimentComponents below) rather than using a separate struct with individual string elements. We are anyway initializing a map and using/converting it.

Reply:
And we could also use it with the `omitempty` option as this may not be a mandatory requirement for other usecases.

Reply:
We could also be specific in naming the attribute. For example, here - in this case - it would be RunnerAnnotation (while both this and the ExperimentAnnotation might be same -i.e., mapped to some app/platform context, we could also leverage the enhancements you are making to embed separate/distinct properties into the respective/individual chaos resources. In your use current usecase, it might be the same annotation being reused for both runner/experiment, in others they could be different)

Reply:
```
RunnerAnnotations map[string]string `json:"runnerannotations"` 
```